platform/web/os_web.cpp

```diff
@@ -151,6 +151,13 @@ bool OS_Web::_check_internal_feature_support(const String &p_feature) {
 	if (godot_js_os_has_feature(p_feature.utf8().get_data())) {
 		return true;
 	}
+
+#ifdef WEB_DLINK_ENABLED
+	if (p_feature == "web_extensions") {
+		return true;
+	}
```

Comment:
I wonder if this should be a generic `gdextension` feature instead, so that it can be enabled on all platforms that support GDExtension at build-time.

If we decide to keep the `web_` prefix, then it should be `web_gdextension`, [similar to `web_macos`, `web_ios` and so on](https://github.com/godotengine/godot/blob/b97110cd307e4d78e20bfafe5de6c082194b2cd6/platform/web/js/libs/library_godot_os.js#L300-L321).

Reply:
I think it could make sense to have a generic `gdextension` feature, if also have `GDEXTENSION_ENABLED`  (or `NOGDEXTENSION`) flag to enable/disable the GDExtension code (though I'm not sure if it's worth it).

In case, we might want to add the feature flag to the export plugin too like done in #93556